env.d/production

```diff
@@ -0,0 +1,27 @@
+# Nota bene
```

Comment:
It's weird to name this file `production` and only use it in the CI. I know it's to fit with what is done in the `docker-compose` file, but it's confusing IMO.

Reply:
I know, this is why I wrote this preamble... I have no better option.